static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.spec.tsx

```diff
@@ -177,4 +186,34 @@ describe('NewWidgetBuiler', function () {
       })
     );
   });
+
+  it('changes the visualization type', async function () {
```

Comment:
Long term, I'd actual prefer if we didn't test individual field changes in the builder as much. I'm still trying to find out the best way to test all of these but if the hook has coverage for the field changes and the fields have their own coverage, that means the widget builder tests shouldn't duplicate that coverage and test the logic that is specifically unique to that component. This should make it easier to test and also faster